docs/getting-started/libnetwork/CalicoSwarm.md

```diff
@@ -0,0 +1,83 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-docker source tree.
+>
+> View the calico-docker documentation for the latest release [here](https://github.com/projectcalico/calico-docker/blob/v0.11.0/README.md).
+<!--- else
+> You are viewing the calico-docker documentation for release **release**.
+<!--- end of master only -->
+
+# Running Calico on a Docker Swarm
+
+The following instructions configure a Docker Swarm cluster networked using Calico. In this tutorial, we will do the following:
+
+* Configure etcd and Calico on a cluster.
+* Configure Docker Swarm on our VM cluster.
+
+## Prerequisites 
+
+This demonstrations assumes that your client and each node in your cluster have `calicoctl`, etcd, and Docker 1.9 or greater installed in your `$PATH`. See our [Environment Setup guide](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/libnetwork/EnvironmentSetup.md) for instructions on getting this properly set up.
+
+## Configure etcd
+
+Calico requires etcd, so let's run it on our cluster. We need to start etcd with the IP address of the manager server, so store the manager server's IP as environment variable for easy access.
+
+    $ export MANAGER_IP=<Manager's IP Address>
+    
+Then, start etcd:
+
+    $ etcd -name etcd0  -advertise-client-urls http://$MANAGER_IP:2379,http://$MANAGER_IP:4001 \
```

Comment:
I don't think we need quite as many start-up parms as we have here.  Can we simplify?